ui_components/src/travel/Map.js

```diff
@@ -123,14 +123,92 @@ export const Map = ({ mapData, scoutData, playerId, ranksToView, strategicView,
+                                        stroke="#592424"
+                                        cx="24.5"
+                                        cy="24"
+                                        r="15"
+                                        strokeWidth="5"
+                                        strokeMiterlimit="0"
+                                        fill="none"
+                                        transform="rotate(-90, 24.5, 24)"
+                                    />
+                                    <circle
+                                        id="operation_progress_circle_background"
+                                        stroke="#592424"
+                                        cx="24.5"
+                                        cy="24"
+                                        r="10"
+                                        strokeWidth="11"
+                                        strokeMiterlimit="0"
+                                        fill="none"
+                                        strokeDasharray="62.83"
+                                        strokeDashoffset="0"
+                                        transform="rotate(-90, 24.5, 24)"
+                                    />
+                                    <circle
+                                        id="operation_progress_circle"
+                                        stroke="#ff6a6a"
+                                        cx="24.5"
+                                        cy="24"
+                                        r="10"
+                                        strokeWidth="5"
+                                        strokeMiterlimit="0"
+                                        fill="none"
+                                        strokeDasharray="62.83"
+                                        strokeDashoffset={62.83 - ((62.83 / 100) * mapData.operation_progress)}
+                                        transform="rotate(-90, 24.5, 24)"
+                                    />
+                                    <circle
+                                        id="operation_interval_circle"
+                                        stroke="#00b044"
+                                        cx="24.5"
+                                        cy="24"
+                                        r="15"
+                                        strokeWidth="2"
+                                        strokeMiterlimit="0"
+                                        fill="none"
+                                        strokeDasharray="100"
+                                        strokeDashoffset={100 - ((100 / 100) * mapData.operation_interval)}
+                                        transform="rotate(-90, 24.5, 24)"
+                                    />
+                                </svg>
+                            </div>
+                            </>
+                        }
+                    </div>
```

Comment:
This should get pulled out into its own component, there's already so much going on in this one

Reply:
Completed.